examples/miral-shell/spinner/eglapp.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2013 Canonical Ltd.
```

Comment:
-2018? And remove Daniel as author, it's a trivial bit of boilerplate.

Reply:
Actually, no. This is just a restored file.